README.md

```diff
@@ -72,8 +72,8 @@ datetime.timedelta(0, 3600)
 You can get a sanitized version of the field given to the constructor with the *sanitize* method or the **field** attribute.
 
 ```python
-# Field : "mo-su 0930-2000"
->>> print(hoh.sanitize())
+>>> field = "mo-su 0930-2000"
+>>> print(hoh.sanitize(field))
 Mo-Su 09:30-20:00
```

Comment:
I forgot that: can you turn this into string (`"Mo-Su 09:30-20:00"` for example), as it returns a string? It remains probably a few like this one.

Thank you!